src/components/dumb/FilterRadio.vue

```diff
@@ -7,7 +7,7 @@
       :name="`radio-${props.id}-${option.value}`"
```

Comment:
hier had ik nog bedenking bij: de name moet toch common zijn tussende radio opties. dan mag de value er toch niet in zitten?

Reply:
Correct, goed gezien